src/app/cluster/details/cluster/template.html

```diff
@@ -340,8 +340,12 @@
                                  [value]="cluster.spec.enableUserSSHKeyAgent">
             </km-property-boolean>
 
+            <km-property-boolean label="External CCM/CSI"
+                                 [value]="cluster.status.externalCCMMigration === 'NotNeeded'">
```

Comment:
There is enum available in the template that can be used instead of hardcoded string. `externalCCMMigrationStatus.NotNeeded`